Auto update failing for weechat

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [weechat](https://github.com/termux/termux-packages/blob/44f7fbe7d2faf5e61664f17eda7a866690c9769f/packages/weechat) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating weechat [Current version: 4.3.5]
INFO: package being updated to 4.3.6.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 34 2631k   34  911k    0     0   559k      0  0:00:04  0:00:01  0:00:03  559k
100 2631k  100 2631k    0     0  1479k      0  0:00:01  0:00:01 --:--:-- 1479k

You are about to commit these changes:

--------------------
diff --git a/packages/weechat/build.sh b/packages/weechat/build.sh
index 020d5d39e..5a98bb27f 100644
--- a/packages/weechat/build.sh
+++ b/packages/weechat/build.sh
@@ -4,9 +4,9 @@ TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # `weechat-python-plugin` depends on libpython${TERMUX_PYTHON_VERSION}.so.
 # Please revbump and rebuild when bumping TERMUX_PYTHON_VERSION.
-TERMUX_PKG_VERSION="4.3.5"
+TERMUX_PKG_VERSION="4.3.6"
 TERMUX_PKG_SRCURL=https://www.weechat.org/files/src/weechat-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=e6dbc4c832da5c5b85e496fefc15237fbbe2a8f7ba2fbe81ee071dc0c66b4be3
+TERMUX_PKG_SHA256=878b06391532ddc40f4b495462a217ebc39925e2eadfe4df84e76a3313647c99
 TERMUX_PKG_DEPENDS="libandroid-support, libcurl, libgcrypt, libgnutls, libiconv, ncurses, zlib, zstd"
 TERMUX_PKG_BREAKS="weechat-dev"
 TERMUX_PKG_REPLACES="weechat-dev"
--------------------

bump(main/weechat): 4.3.6


Not committing to Git!

INFO: Trying to build package.
INFO: Package weechat exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building weechat for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   143k      0 --:--:-- --:--:-- --:--:--  144k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1461      0 --:--:-- --:--:-- --:--:--  1461
100   833  100   833    0     0   1461      0 --:--:-- --:--:-- --:--:--  1461
gpg: Signature made Thu 15 Aug 2024 06:21:52 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  1 1615k    1 30623    0     0  36991      0  0:00:44 --:--:--  0:00:44 36984
100 1615k  100 1615k    0     0  1124k      0  0:00:01  0:00:01 --:--:-- 1124k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  24539      0 --:--:-- --:--:-- --:--:-- 24530
100 13320  100 13320    0     0  24533      0 --:--:-- --:--:-- --:--:-- 24530
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1523      0 --:--:-- --:--:-- --:--:--  1525
gpg: Signature made Tue 13 Aug 2024 05:58:27 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  24363      0 --:--:-- --:--:-- --:--:-- 24341
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1507      0 --:--:-- --:--:-- --:--:--  1509
gpg: Signature made Thu 15 Aug 2024 12:33:54 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.62.1 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp3@1.4.0 if necessary...
Found libnghttp3 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnghttp3/libnghttp3_1.4.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 64044  100 64044    0     0   781k      0 --:--:-- --:--:-- --:--:--  791k
extracting libnghttp3 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.07.02 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.07.02 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.3.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.0 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libssh2/libssh2_1.11.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  193k  100  193k    0     0  2136k      0 --:--:-- --:--:-- --:--:-- 2146k
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.9.1 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcurl/libcurl_8.9.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  869k  100  869k    0     0  7738k      0 --:--:-- --:--:-- --:--:-- 7767k
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.9.1 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/curl/curl_8.9.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 27  203k   27 56665    0     0   641k      0 --:--:-- --:--:-- --:--:--  636k
100  203k  100  203k    0     0  1955k      0 --:--:-- --:--:-- --:--:-- 1953k
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgpg-error@1.50 if necessary...
Found libgpg-error in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgpg-error/libgpg-error_1.50_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  111k  100  111k    0     0  1375k      0 --:--:-- --:--:-- --:--:-- 1378k
extracting libgpg-error to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgcrypt@1.10.3-2 if necessary...
Found libgcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgcrypt/libgcrypt_1.10.3-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  428k  100  428k    0     0  3640k      0 --:--:-- --:--:-- --:--:-- 3659k
extracting libgcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@27 if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgmp@6.3.0 if necessary...
Found libgmp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgmp/libgmp_6.3.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  315k  100  315k    0     0  3139k      0 --:--:-- --:--:-- --:--:-- 3150k
extracting libgmp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  543k  100  543k    0     0  5677k      0 --:--:-- --:--:-- --:--:-- 5719k
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8988  100  8988    0     0  84445      0 --:--:-- --:--:-- --:--:-- 84792
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libunistring@1.1 if necessary...
Found libunistring in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libunistring/libunistring_1.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  508k  100  508k    0     0  4118k      0 --:--:-- --:--:-- --:--:-- 4132k
extracting libunistring to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libidn2@2.3.7 if necessary...
Found libidn2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libidn2/libidn2_2.3.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  102k  100  102k    0     0  1199k      0 --:--:-- --:--:-- --:--:-- 1209k
extracting libidn2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnettle@3.9.1 if necessary...
Found libnettle in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnettle/libnettle_3.9.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  393k  100  393k    0     0  3444k      0 --:--:-- --:--:-- --:--:-- 3450k
extracting libnettle to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency nettle@3.9.1 if necessary...
Found nettle in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/nettle/nettle_3.9.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 18208  100 18208    0     0   228k      0 --:--:-- --:--:-- --:--:--  230k
extracting nettle to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libevent@2.1.12-2 if necessary...
Found libevent in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libevent to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.6.2 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4.20231001-5 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4.20231001-5 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4.20231001-5 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4.20231001-5 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.24 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.6-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.2 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.2 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.46.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.46.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.46.1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.43 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.15.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.15.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.1 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.10 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.11.9-5 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.11.9-5 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.11.9-5 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency resolv-conf@1.3 if necessary...
Found resolv-conf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting resolv-conf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.44 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.44 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency swig@4.2.1 if necessary...
Found swig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting swig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libunbound@1.21.0 if necessary...
Found libunbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libunbound/libunbound_1.21.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  349k  100  349k    0     0  3499k      0 --:--:-- --:--:-- --:--:-- 3531k
extracting libunbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency unbound@1.21.0 if necessary...
Found unbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/u/unbound/unbound_1.21.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  571k  100  571k    0     0  5728k      0 --:--:-- --:--:-- --:--:-- 5768k
extracting unbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pyunbound@1.21.0 if necessary...
Found pyunbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pyunbound/pyunbound_1.21.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 38292  100 38292    0     0  42650      0 --:--:-- --:--:-- --:--:-- 42641
extracting pyunbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgnutls@3.8.5 if necessary...
Found libgnutls in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgnutls/libgnutls_3.8.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  711k  100  711k    0     0  6360k      0 --:--:-- --:--:-- --:--:-- 6405k
extracting libgnutls to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gnutls@3.8.5 if necessary...
Found gnutls in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gnutls/gnutls_3.8.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  172k  100  172k    0     0  1659k      0 --:--:-- --:--:-- --:--:-- 1670k
extracting gnutls to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblua53@5.3.6 if necessary...
Found liblua53 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblua53/liblua53_5.3.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 84444  100 84444    0     0   950k      0 --:--:-- --:--:-- --:--:--  958k
extracting liblua53 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency lua53@5.3.6 if necessary...
Found lua53 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/l/lua53/lua53_5.3.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 68 88512   68 60902    0     0  68630      0  0:00:01 --:--:--  0:00:01 68583
100 88512  100 88512    0     0  98777      0 --:--:-- --:--:-- --:--:-- 98785
extracting lua53 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-utimes@0.4 if necessary...
Found libandroid-utimes in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-utimes/libandroid-utimes_0.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  3120  100  3120    0     0  46629      0 --:--:-- --:--:-- --:--:-- 47272
extracting libandroid-utimes to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency perl@5.38.2-3 if necessary...
Found perl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/perl/perl_5.38.2-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13.5M  100 13.5M    0     0  71.3M      0 --:--:-- --:--:-- --:--:-- 71.5M
extracting perl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-execinfo@0.1-1 if necessary...
Found libandroid-execinfo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-execinfo/libandroid-execinfo_0.1-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14192  100 14192    0     0   169k      0 --:--:-- --:--:-- --:--:--  171k
extracting libandroid-execinfo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libyaml@0.2.5-4 if necessary...
Found libyaml in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liby/libyaml/libyaml_0.2.5-4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 43774  100 43774    0     0   598k      0 --:--:-- --:--:-- --:--:--  602k
extracting libyaml to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ruby@3.3.2 if necessary...
Found ruby in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/ruby/ruby_3.3.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7193k  100 7193k    0     0  41.2M      0 --:--:-- --:--:-- --:--:-- 41.0M
extracting ruby to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ruby-ri@3.3.2 if necessary...
Found ruby-ri in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/ruby-ri/ruby-ri_3.3.2_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  3 1790k    3 63378    0     0  64416      0  0:00:28 --:--:--  0:00:28 64408
100 1790k  100 1790k    0     0  1126k      0  0:00:01  0:00:01 --:--:-- 1126k
extracting ruby-ri to /home/builder/.termux-build/_cache-all...
Downloading dependency zstd@1.5.6 if necessary...
Found zstd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/z/zstd/zstd_1.5.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  329k  100  329k    0     0  3438k      0 --:--:-- --:--:-- --:--:-- 3465k
extracting zstd to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pzstd@1.5.6 if necessary...
Found pzstd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pzstd/pzstd_1.5.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 40540  100 40540    0     0   540k      0 --:--:-- --:--:-- --:--:--  542k
extracting pzstd to /home/builder/.termux-build/_cache-aarch64...
Downloading https://www.weechat.org/files/src/weechat-4.3.6.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  3 2631k    3 81516    0     0  75317      0  0:00:35  0:00:01  0:00:34 75338
100 2631k  100 2631k    0     0  1460k      0  0:00:01  0:00:01 --:--:-- 1461k
Applying patch: cmake-FindPerl.cmake.patch
Applying patch: no-setuid.patch
Applying patch: src-core-core-config.c.patch
Applying patch: src-core-core-url.c.patch
Applying patch: src-core-hook-hook-url.c.patch
Applying patch: src-core-hook-hook-url.h.patch
Applying patch: src-core-weechat.c.patch
Applying patch: src-gui-curses-CMakeLists.txt.patch
Downloading https://github.com/Kitware/CMake/releases/download/v3.29.3/cmake-3.29.3-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 50.7M  100 50.7M    0     0  67.4M      0 --:--:-- --:--:-- --:--:-- 67.4M
Downloading https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  120k  100  120k    0     0   290k      0 --:--:-- --:--:-- --:--:--  290k
Archive:  /home/builder/.termux-build/weechat/tmp/ninja-1.12.1.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.12.1/ninja  
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 18.0.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r27-api-24-v0/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found PkgConfig: /home/builder/.termux-build/_cache/android-r27-api-24-v0/bin/pkg-config (found version "0.29.2")
-- Looking for include file langinfo.h
-- Looking for include file langinfo.h - found
-- Looking for include file sys/resource.h
-- Looking for include file sys/resource.h - found
-- Looking for include file malloc.h
-- Looking for include file malloc.h - found
-- Looking for malloc_trim
-- Looking for malloc_trim - not found
-- Looking for mallinfo
-- Looking for mallinfo - found
-- Looking for mallinfo2
-- Looking for mallinfo2 - not found
-- Looking for eat_newline_glitch
-- Looking for eat_newline_glitch - found
-- Checking for module 'libgcrypt'
--   Found libgcrypt, version 1.10.3-unknown
-- Found ZLIB: /data/data/com.termux/files/usr/lib/libz.so (found version "1.3.1")
-- Checking for module 'libzstd'
--   Found libzstd, version 1.5.6
-- Looking for libiconv_open in /data/data/com.termux/files/usr/lib/libiconv.so
-- Looking for libiconv_open in /data/data/com.termux/files/usr/lib/libiconv.so - found
-- Looking for iconv_open in /data/data/com.termux/files/usr/lib/libiconv.so
-- Looking for iconv_open in /data/data/com.termux/files/usr/lib/libiconv.so - not found
-- Performing Test ICONV_2ARG_IS_CONST
-- Performing Test ICONV_2ARG_IS_CONST - Failed
-- Found CURL: /data/data/com.termux/files/usr/lib/libcurl.so (found version "8.9.1")
-- Looking for include file libintl.h
CMake Warning (dev) at /home/builder/.termux-build/_cache/cmake-3.29.3/share/cmake-3.29/Modules/CheckIncludeFiles.cmake:116 (message):
  Policy CMP0075 is not set: Include file check macros honor
  CMAKE_REQUIRED_LIBRARIES.  Run "cmake --help-policy CMP0075" for policy
  details.  Use the cmake_policy command to set the policy and suppress this
  warning.

  CMAKE_REQUIRED_LIBRARIES is set to:

    /data/data/com.termux/files/usr/lib/libiconv.so

  For compatibility with CMake 3.11 and below this check is ignoring it.
Call Stack (most recent call first):
  cmake/FindGettext.cmake:46 (check_include_files)
  CMakeLists.txt:262 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Looking for include file libintl.h - found
-- Looking for dgettext
-- Looking for dgettext - not found
-- Found Intl: built in to C library
-- Looking for flock
-- Looking for flock - found
-- Looking for backtrace
-- Looking for backtrace - not found
-- Checking for module 'python3-embed'
--   Found python3-embed, version 3.11
-- Checking for one of the modules 'ruby-3.3;ruby-3.2;ruby-3.1;ruby-3.0;ruby-2.7;ruby-2.6;ruby-2.5;ruby-2.4;ruby-2.3;ruby-2.2;ruby-2.1;ruby-2.0;ruby-1.9;ruby'
-- Checking for one of the modules 'lua5.4;lua-5.4;lua54;lua5.3;lua-5.3;lua53;lua5.2;lua-5.2;lua52;lua5.1;lua-5.1;lua51;lua-5.0;lua5.0;lua50;lua'
-- Checking for one of the modules 'ncursesw'
-- Looking for include file ncursesw/ncurses.h
-- Looking for include file ncursesw/ncurses.h - found
-- Configuring done (1.7s)
CMake Warning at src/plugins/perl/CMakeLists.txt:43 (target_link_libraries):
  Target "perl" requests linking to directory
  "/data/data/com.termux/files/usr/lib".  Targets may link only to libraries.
  CMake is dropping the item.


-- Generating done (0.1s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_INSTALL_LIBDIR
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/weechat/build
[1/263] cd /home/builder/.termux-build/weechat/build && /home/builder/.termux-build/weechat/src/tools/set_git_version.sh /home/builder/.termux-build/weechat/src 4.3.6 config-git.h
Creating file config-git.h with git version: ""
[2/263] Building C object src/core/CMakeFiles/weechat_core.dir/weechat.c.o
[3/263] Building C object src/plugins/CMakeFiles/weechat_plugins.dir/plugin-api.c.o
[4/263] Building C object src/core/CMakeFiles/weechat_core.dir/core-backtrace.c.o
[5/263] Building C object src/core/CMakeFiles/weechat_core.dir/core-arraylist.c.o
[6/263] Building C object src/plugins/CMakeFiles/weechat_plugins.dir/plugin-api-info.c.o
[7/263] Building C object src/core/CMakeFiles/weechat_core.dir/core-calc.c.o
[8/263] Building C object src/plugins/CMakeFiles/weechat_plugins.dir/plugin.c.o
[9/263] Building C object src/core/CMakeFiles/weechat_core.dir/core-completion.c.o
[10/263] Building C object src/core/CMakeFiles/weechat_core.dir/core-crypto.c.o
FAILED: src/core/CMakeFiles/weechat_core.dir/core-crypto.c.o 
/home/builder/.termux-build/_cache/android-r27-api-24-v0/bin/clang --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27-api-24-v0 --sysroot=/home/builder/.termux-build/_cache/android-r27-api-24-v0/sysroot -DENABLE_NLS -DHAVE_CONFIG_H -DHAVE_GCRYPT -DHAVE_ICONV -DHAVE_ZSTD -DWEECHAT_LICENSE=\"GPL3\" -DWEECHAT_VERSION=\"4.3.6\" -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -D_LARGE_FILES -I/home/builder/.termux-build/weechat/build -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -fsigned-char -fms-extensions -Wall -Wextra -Werror-implicit-function-declaration -Wformat -Werror=format-security -O3 -DNDEBUG -fPIC -MD -MT src/core/CMakeFiles/weechat_core.dir/core-crypto.c.o -MF src/core/CMakeFiles/weechat_core.dir/core-crypto.c.o.d -o src/core/CMakeFiles/weechat_core.dir/core-crypto.c.o -c /home/builder/.termux-build/weechat/src/src/core/core-crypto.c
/home/builder/.termux-build/weechat/src/src/core/core-crypto.c:527:29: error: call to undeclared function 'htobe64'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  527 |     moving_factor_swapped = htobe64 (moving_factor);
      |                             ^
1 error generated.
[11/263] Building C object src/core/CMakeFiles/weechat_core.dir/core-config-file.c.o
[12/263] Building C object src/core/CMakeFiles/weechat_core.dir/core-config.c.o
[13/263] Building C object src/core/CMakeFiles/weechat_core.dir/core-command.c.o
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-08-15 18:23:11 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/10408324102">10408324102</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/weechat/build.sh
+++ b/packages/weechat/build.sh
@@ -4,9 +4,9 @@ TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # `weechat-python-plugin` depends on libpython${TERMUX_PYTHON_VERSION}.so.
 # Please revbump and rebuild when bumping TERMUX_PYTHON_VERSION.
-TERMUX_PKG_VERSION="4.3.5"
+TERMUX_PKG_VERSION="4.3.6"
 TERMUX_PKG_SRCURL=https://www.weechat.org/files/src/weechat-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=e6dbc4c832da5c5b85e496fefc15237fbbe2a8f7ba2fbe81ee071dc0c66b4be3
+TERMUX_PKG_SHA256=878b06391532ddc40f4b495462a217ebc39925e2eadfe4df84e76a3313647c99
 TERMUX_PKG_DEPENDS="libandroid-support, libcurl, libgcrypt, libgnutls, libiconv, ncurses, zlib, zstd"
 TERMUX_PKG_BREAKS="weechat-dev"
 TERMUX_PKG_REPLACES="weechat-dev"

--- a/packages/weechat/src-core-core-crypto.c.patch
+++ b/packages/weechat/src-core-core-crypto.c.patch
@@ -1,0 +1,16 @@
+diff -u -r ../weechat-4.3.6/src/core/core-crypto.c ./src/core/core-crypto.c
+--- ../weechat-4.3.6/src/core/core-crypto.c	2024-08-15 13:21:34.000000000 +0000
++++ ./src/core/core-crypto.c	2024-08-15 19:00:04.048492591 +0000
+@@ -31,6 +31,12 @@
+ #include <math.h>
+ #include <gcrypt.h>
+ 
++#ifdef __ANDROID__
++/* Bring in htobe64 */
++#define _BSD_SOURCE
++#include <endian.h>
++#endif
++
+ #include "weechat.h"
+ #include "core-crypto.h"
+ #include "core-config-file.h"

